opba-banking-protocol-facade/src/test/java/de/adorsys/opba/protocol/facade/services/authorization/ServiceSessionAndAuthSessionTest.java

```diff
@@ -0,0 +1,244 @@
+        authData.put("psuId", "anton.brueckner");
+        authData.put("consent.validUntil", "2020-02-26T22:00:00.000Z");
+        authData.put("consent.access.allAccountsAccess", "allAccountsWithBalances");
+
+        return AuthorizationRequest.builder()
+                       .facadeServiceable(FacadeServiceableRequest.builder()
+                                                  .redirectCode(listAccountsResponse.getRedirectCode())
+                                                  .authorizationSessionId(listAccountsResponse.getAuthorizationSessionId())
+                                                  .requestId(listAccountsResponse.getXRequestId())
+                                                  .build()
+                       )
+                       .scaAuthenticationData(authData)
+                       .build();
+    }
+
+    private ListAccountsRequest buildListAccountRequest(UUID sessionId) {
+        return ListAccountsRequest.builder()
+                       .facadeServiceable(
+                               FacadeServiceableRequest.builder()
+                                       .bankId(TEST_BANK_ID)
+                                       .requestId(REQUEST_ID)
+                                       .serviceSessionId(sessionId)
+                                       .sessionPassword(PASSWORD)
+                                       .fintechRedirectUrlOk(REDIRECT_URL_OK)
+                                       .fintechRedirectUrlNok(REDIRECT_URL_NO_OK)
+                                       .build()
+                       ).build();
+    }
+
+    private void assertAllSessions(UUID sessionId) {
+        Optional<AuthSession> authenticationSessionOptional = authenticationSessions.findById(sessionId);
+        Optional<ServiceSession> serviceSessionOptional = serviceSessionRepository.findById(sessionId);
+
+        assertThat(authenticationSessionOptional.isPresent()).isTrue();
+        assertThat(serviceSessionOptional.isPresent()).isTrue();
+
+        AuthSession authenticationSession = authenticationSessionOptional.get();
+        ServiceSession serviceSessionFromAuth = authenticationSession.getParent();
+        ServiceSession serviceSessionFromDB = serviceSessionOptional.get();
+
+        assertThat(serviceSessionFromDB.getId()).isEqualTo(serviceSessionFromAuth.getId());
+        assertThat(serviceSessionFromDB.getAuthSession().getId()).isEqualTo(authenticationSession.getId());
+        assertThat(serviceSessionFromDB.getAuthSession().getRedirectCode()).isEqualTo(authenticationSession.getRedirectCode());
+    }
+
+    private AuthorizationRequiredResult buildAuthorizationRequiredResult() {
+        return new AuthorizationRequiredResult(URI.create("http://localhost:4400/account-information/login?encryptedConsentId=ZwaFosuf-GopQSJwEcWKIn708zVNbrHvmWZh6amuM59aBEMABLWcMZATy6v9CghY5hpqT3HlUQAcRciH8hB3asz9MpaJIQIH3NJX8IHgetw=_=_psGLvQpt9Q&redirectId=7154aa62-b744-4072-b4b9-0fa87048e2c8"));
```

Comment:
Makes no sense to embed this large URL 
`http://localhost:4400/account-information/`
is completely enough